### ISSUE TITLE
Can u write  a private version for me ？

### ISSUE THREAD

**Issue by jacnk97** [Jire/Charlatano#507]
I want to use ur cheat to play 5ewin , so i want a private version and i'll pay 500-1000 dollars for a mounth .

**Comment by w-o-a-h** [comment]
what is 5ewin

**Comment by hsari26** [comment]
YEAH, this cheat is amazing for me but i want to league mode only like ESEA/FACEIT with client ofcourse.I want to pay it.Can you  write private for me too.I can give you  25-30 dollar per month.I want to only aimlock or esp.But esea take screenshot i dont know.Aimlock good for me.

**Comment by jacnk97** [comment]
@w-o-a-h 5ewin is an esports platform in China and Asia ，5ewin anti-cheats like esea it inject a cheano.dll to the csgo 
https://www.5ewin.com/

**Comment by jacnk97** [comment]
@hsari26 no i just want a hack for 5ewin ,not esea or faceit
